### ISSUE TITLE
Stuttering / pixelation 

### ISSUE THREAD

**Issue by fyrstyk** [catalinii/minisatip#358]
Hi

I've managed to compile your code on a Vu+Duo2, and I've managed to stream, but I'm getting issues with pixellation and stuttering audio.

Here is a log, can you advise? I'd love to get this working :)

[log.txt](https://github.com/catalinii/minisatip/files/981736/log.txt)

**Comment by catalinii** [comment]
Hi,

what client do you use ?
change:

#define TCP_RTP_CHUNKS 20
to

#define TCP_RTP_CHUNKS 5

in stream.h and tell me if it makes a difference.

Thank you

**Comment by fyrstyk** [comment]
I'm using the Elgato SAT>IP client on my Android mobile, and also TVHeadend. I've been having issues with TVHeadend and discovery of muxes, so decided to use a standalone SAT>IP app for testing. The issues with stuttering / breakup are only on the mobile, on TVHeadend I can only tune 1 channel. 

Minisatip is on my VU+ Duo2 box, and I have confirmed that there are no issues with the incoming signal by tuning on the box and viewing direct. I then un-configure all the tuners so that they are available for ministatip.

I've made the changes, recompiled, will test when I get home, and will provide an update.

Thanks :)
